runtime/src/main/java/org/corfudb/util/serializer/DynamicProtobufSerializer.java

```diff
@@ -344,6 +344,13 @@ public DynamicMessage createDynamicMessageFromJson(Any anyMsg,
         return builder.build();
     }
 
+    public DynamicMessage createDynamicMessage(Any message) throws DescriptorValidationException, InvalidProtocolBufferException {
+        String fullMessageName = getFullMessageName(message);
```

Comment:
Isn't class method like createDynamicMessage can access private methods of the same class? You can just make createDynamicMessage public and leave the other method private? Or do you need to access these methods outside of this class too?

Reply:
yes some of these other methods are useful to have too - so I thought it is best to make them public as they offer functionality.